packages/field-group-react/src/FieldGroup.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { ReactNode } from "react";
+
+interface Props {
+    legend: string;
+    children?: ReactNode;
+    className?: string;
+}
+
+export const FieldGroup = ({ legend, className, children }: Props) => (
```

Comment:
Do we really need this FieldGroup? I don't feel it creates enough value to a be a separate react-component? It feels too DRY.

Reply:
If we look at CheckboxGroup now it doesn't do anything. It's just a Component that wraps a Component. Or a function that calls another function.

Reply:
I would flip it around and ask, do we really need the CheckboxGroup? We have a couple of alternatives:
- We can have the same API in CheckboxGroup and RadioButtonChoice, and implement the same wrapper and style twice
- We can have different APIs, but otherwise same as above
- We can have a separate FieldGroup with style and structure and use it standalone and in other components

I don't think the first option is right, since the resulting code is not DRY and these are two quite different concepts: With radio buttons you choose _between_ options, and as such the choices can be considered as one thing (and in fact _are_ considered one thing in standard HTML forms). With checkboxes however, the boxes each represent one choice, and the group is there to visually and semantically connect them.

The second option.. 🤷‍♂️ 

The third option is how it is implemented here. I agree that the CheckboxGroup might be a too thin wrapper around FieldGroup to be considered useful, so maybe it is not needed?